### DISCUSSION TITLE
Quick size and position buttons on player

### DISCUSSION THREAD

**Discussion by bruzie** [f1multiviewer/issue-tracker#53] [Feature requests]
What I think would be handy would be some quick size and position buttons on hover for the video player, so that we don't have to go through the right-click menu and navigate through the tree. Only really need thirds and quarters (with 1/2s and 2/3s).

The position icon could just be a 3x3 or 4x4 grid and you just select a square to position.

**Comment by f1multiviewer** [comment]
Totally, at some point everything should be part of the UI, but especially on Windows there are a number of issues with dragging the Window and the internals of how draggable areas "eat up" all mouse events that make this a bit more difficult. It's planned though (as stated in the docs)

**Reply by bruzie** [reply]
If the UI is a pain for this, maybe hotkeys, e.g. ALT+SHIFT+9 to move to top-right, etc.?
